api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -448,21 +450,20 @@ public void updateDbRdrExport(RdrEntity entity, List<Long> idList) {
   }
 
   /**
-   * Set excludeFromPublicDirectory to true if the workspace creator is an operational user i.e has
-   * Institution as All of Us Program operational Use
+   * Should return true if the creator does not have verified institution or if the user is an
+   * Operational User (verified institution is AouOps)
    *
    * @param creatorUser
-   * @param rdrWorkspace
+   * @return
    */
-  void setExcludeFromPublicDirectory(DbUser creatorUser, RdrWorkspace rdrWorkspace) {
-    rdrWorkspace.setExcludeFromPublicDirectory(false);
-    verifiedInstitutionalAffiliationDao
-        .findFirstByUser(creatorUser)
-        .ifPresent(
-            verifiedInstitutionalAffiliation -> {
-              rdrWorkspace.setExcludeFromPublicDirectory(
-                  institutionService.validateOperationalUser(
-                      verifiedInstitutionalAffiliation.getInstitution()));
-            });
+  boolean isExcludeFromPublicDirectory(DbUser creatorUser) {
+    Optional<DbVerifiedInstitutionalAffiliation> dbVerifiedInstitutionalAffiliation =
```

Comment:
We can keep this in a functional/streaming style.  Try this:

```
return verifiedInstitutionalAffiliationDao
    .findFirstByUser(creatorUser)
    .map(institutionService::validateOperationalUser)
    .orElse(true);
```

Looks like that will require updating the signature of `validateOperationalUser()` but that seems worthwhile to me.

Reply:
Also, please rename the method from `validateOperationalUser` to `isOperationalUser` so we know it's a predicate method. 

I'm not 100% sure about the `.orElse(true)` if that's covering up an error state we're interested in. Maybe `orElseThrow()` or `orElseGet` with a log statement.

Reply:
null is not an error state here.  not found indicates "no affiliation" which is the logic we are accounting for in this change.